manifests/kots-app.yaml

```diff
@@ -23,6 +23,8 @@ spec:
     - deployment/carto-workspace-subscriber
     - deployment/carto-workspace-www
   minKotsVersion: 1.107.4
+  additionalImages:
+    - gcr.io/carto-onprem-artifacts/tenant-requirements-checker:2024.4.17-rc.7
```

Comment:
I've already asked Replicated team why the bundle is not automatically adding this image in it: https://cartoteam.slack.com/archives/C05V5V84W7M/p1718285246347899 For the moment we need this for Verizon